docs/docs/self-hosting/manage/cli/mirror.mdx

```diff
@@ -214,11 +210,12 @@ Prints the amount of rows of the source and destination database and the diff. P
 
 The following tables will likely have an unequal count:
 
-* **eventstore.events2**: because only event of instances are migrated
-* **projections.current_states**: If your deployment was upgraded several times it the number of entries in the destination will be lower
-* **projections.locks**: If your deployment was upgraded several times it the number of entries in the destination will be lower
-* **projections.keys4\***: Only not expired keys are inserted
+* **projections.current_states**: If your deployment was upgraded several times, the number of entries in the destination will be lower
+* **projections.locks**: If your deployment was upgraded several times, the number of entries in the destination will be lower
+* **projections.keys4\***: Only not expired keys are inserted, the number of entries in the destination will be lower
 * **projections.failed_events**: Should be lower or equal.
+* **auth.users2**: Was replaced with auth.users3, the number of entries in the destination will be 0
+* **auth.users3**: Is the replacement of auth.users2, the number of entries in the destination will be equal or higher
```

Comment:
auth.users3 is not likely to have a diff, right?

Reply:
no. it would only have a diff if you mirror an older version which i would not explicitely describe